src/main/resources/listLookup.json

```diff
@@ -10,15 +10,25 @@
         "defaultSensitivity": ""
     },
     "SJP_PRESS_LIST": {
-        "friendlyName": "Single Justice Procedure Press List",
-        "welshFriendlyName": "Rhestr y Wasg Y Weithdrefn Un Ynad",
-        "shortenedFriendlyName": "SJP Press List",
+        "friendlyName": "Single Justice Procedure Press List (Full List)",
+        "welshFriendlyName": "Rhestr y Wasg Y Weithdrefn Un Ynad (Rhestr Lawn)",
+        "shortenedFriendlyName": "SJP Press List (Full list)",
         "url": "sjp-press-list",
         "jurisdictions": ["Single Justice Procedure"],
         "welshJurisdictions": ["Gweithdrefn Un Ynad"],
         "restrictedProvenances": ["PI_AAD"],
         "defaultSensitivity": "CLASSIFIED"
     },
+    "SJP_DELTA_PRESS_LIST": {
+        "friendlyName": "Single Justice Procedure Press List (New Cases)",
+        "welshFriendlyName": "Rhestr y Wasg Y Weithdrefn Un Ynad (Achosion Newydd)",
+        "shortenedFriendlyName": "SJP Press List (New cases)",
+        "url": "sjp-press-list-new-cases",
+        "jurisdictions": ["Single Justice Procedure"],
+        "welshJurisdictions": ["Gweithdrefn Un Ynad"],
```

Comment:
Nothing wrong here but be aware that the jurisdiction for SJP press list has been updated to 'Magistrates' in PUB-1698. So if that PR is merged first, we need to update this to Magistrates too

Reply:
This has now been updated to 'Magistrates'